src/mono/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.xml

```diff
@@ -651,5 +648,9 @@
 			<method name="ApplyUpdateSdb" />
 		</type>
 
+		<type fullname="System.ValueType">
+			<!-- https://github.com/dotnet/runtime/issues/47909 -->
```

Comment:
I don't understand why this issue is attached to this entry. The issue says

> Remove DynamicDependency from ValueType in browser builds

But this entry is unconditionally preserving the constructor on `ValueType`, which are two different things in my mind.

Reply:
I can remove it from the xml if you want. It's only a reminder to remove it once we land final solution for `System.Runtime.InteropServices.JavaScript.Runtime` dependency.

Reply:
Can you explain why we need `ValueType.ctor()` in the xml?

Reply:
We have some tiny sample which managed to not require ValueType hence dropping the required dependency.